Contributing.md

```diff
@@ -0,0 +1,88 @@
+# Contributing
+When contributing to this repository, please first discuss the change you wish to make via issue, email, or any other method with the owners of this repository before making a change.
+Please note we have a code of conduct, please follow it in all your interactions with the project.
+
+# Pull Request Process
+1.	Fork the repository and create a separate branch regarding your contribution.
+2.	Update the Changelog file (Changes to README file should be commited **ONLY** when there are changes relevant to the essence of the project, or the ways to deploy it!) with details of changes to the interface, this includes new environment variables, exposed ports, useful file locations, and container parameters.
```

Comment:
Please move the note out of the parentheses _(Changes to README file should be commited **ONLY** when there are changes relevant to the essence of the project, or the ways to deploy it!)_.